serpent-moltres/full-core/extract-convert.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env python3
+# This file is based in $MOLTRES/python/extractSerpent2GCs.py
+import os
+import numpy as np
+import argparse
+import subprocess
+import serpentTools as sT
+
+
+def makePropertiesDir(outdir, filebase, mapFile, unimapFile, ngroups='3'):
+    '''
+    Takes in a mapping from universe names to material properties.
+    It collapses the material properties from 26 to ngroups.
+    Then, it makes a material properties directory.
+
+    Parameters:
+    -----------
+    outdir: [string]
+        name of directory to write properties to
+    filebase: [string]
+        file base name to give moltres
+    mapFile: [string]
+        file that maps branches to temperatures
+    unimapFile: [string]
+        maps material names to serpent universe
+    ngroups: [string]
+        the options are 3, 6, 9, 12, 15, 15b, 15c, 15d, 15e, 18, 18c, 21
+    '''
+
+    # the constants moltres looks for:
+    goodStuff = ['BETA_EFF', 'Chit', 'Chid', 'lambda', 'Diffcoef', 'Kappa',
```

Comment:
none of this is PEP8 compatible. It may be compatible with the MOOSE python style, which we must follow for a MOOSE app, it is a bit annoying to violate PEP8 for all python files... where is the limit? Just MOOSE related files? Just this repo? Every repo in the group? I strongly prefer PEP8 unless this is going to end up in the MOOSE test harness.